packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/nodeexpress/integrationsteps.md

```diff
@@ -0,0 +1,63 @@
+### 1: Create and display sign-in page
+
+Create a sign-in page that captures the user's username.
+
+<div class="half border">
+
+![Screenshot showing a sign-in page with a username field and a Next button.](/img/pwd-optional/pwd-optional-sign-in-page.png)
+
+</div>
+
+### 2: Submit username
+
+When the user submits their username, call `OktaAuth.idx.authenticate()` passing in the username.
```

Comment:
```suggestion
Call `OktaAuth.idx.authenticate()` passing in the username to submit it to Identity Engine.
```

Reply:
Two things here:

1. I think the "When" clause is needed to give context
2. After discussions with Maurice today, I think we need to get further clarification on how to use the Identity Engine term.

Reply:
REMOVE WHEN ... clause